src/third-party/EnsController.ts

```diff
@@ -0,0 +1,131 @@
+import BaseController, { BaseConfig, BaseState } from '../BaseController';
+import { isValidAddress, toChecksumAddress } from 'ethereumjs-util';
+
+/**
+ * @type EnsEntry
+ *
+ * ENS entry representation
+ *
+ * @property chainId - Id of the associated chain
+ * @property ensName - The ENS name
+ * @property address - Hex address with the ENS name
+ */
+export interface EnsEntry {
+	chainId: number;
```

Comment:
chainId is a number here but a string in line 27 - we should probably keep this consistent, i think it's a string in the extension

Reply:
resolved: `chainId` changed to string everywhere

It's also a string in the extension, at least for the most part.